pkg/bmc/redfish.go

```diff
@@ -133,11 +137,14 @@ func (a *redfishAccessDetails) BuildBIOSSettings(firmwareConfig *metal3v1alpha1.
 }
 
 // iDrac Redfish Overrides
-
 func (a *redfishiDracAccessDetails) Driver() string {
 	return "idrac"
 }
 
+func (a *redfishiDracAccessDetails) BIOSInterface() string {
+	return ""
```

Comment:
It seems like users would probably expect us to be using `idrac-redfish` here also. IIUC it will default to `idrac-wsman` right now?

Reply:
Yes, good point. Fixed.